packages/ui/src/app/pages/Proposals/ProposalPreview.tsx

```diff
@@ -195,6 +195,8 @@ export const ProposalPreview = () => {
             <RowGapBlock gap={16}>
               <Label>Proposer</Label>
               <MemberInfo member={proposal.proposer} />
+              <Label>Stake</Label>
+              {constants ? <TokenValue value={constants.requiredStake} /> : 'Loading...'}
             </RowGapBlock>
```

Comment:
It's a really small issue but currently the spacing looks off ("STAKE" is closer to the proposer than the stake value):
```suggestion
            <RowGapBlock gap={16}>
              <Label>Proposer</Label>
              <MemberInfo member={proposal.proposer} />
            </RowGapBlock>

            <RowGapBlock gap={16}>
              <Label>Stake</Label>
              {constants ? <TokenValue value={constants.requiredStake} /> : 'Loading...'}
            </RowGapBlock>
```